Assets/__Scripts/Beatmap/V2/V2Difficulty.cs

```diff
@@ -301,13 +303,12 @@ private static void LoadCustom(V2Difficulty map, JSONNode mainNode)
                                     foreach (JSONNode n in cNode) bookmarksList.Add(new V2Bookmark(n));
                                     break;
                                 case "_customEvents":
-                                    foreach (JSONNode n in cNode) customEventsList.Add(new V2CustomEvent(n));
+                                    foreach (JSONNode n in cNode) try { customEventsList.Add(new V2CustomEvent(n)); } catch (Exception e) { }
```

Comment:
are there exceptions happening here? should probably fix the error at hand rather than silently swallowing it.

otherwise, please format this into multiple lines ty